pkg/kubernetes/informers.go

```diff
@@ -144,6 +146,29 @@ func (im *InformerManager) AddPodListener(add func(obj interface{}), update func
 	})
 }
 
+// AddFSSListener hooks up add, update, delete callbacks for cnscsisvfeaturestate CR
+func (im *InformerManager) AddFSSListener(ctx context.Context, crdGroup, crdVersion, crdName, namespace string,
+	add func(obj interface{}), update func(oldObj, newObj interface{}), remove func(obj interface{})) error {
+	log := logger.GetLogger(ctx)
+	if im.fssInformer == nil {
+		var err error
+		im.fssInformer, err = GetDynamicInformer(ctx, crdGroup, crdVersion, crdName, namespace)
+		if err != nil {
+			log.Errorf("Failed to create a dynamic informer for %s CR. Error - %+v", crdName, err)
```

Comment:
nit: `Error -` to `Error:` for consistency

Reply:
Done

Reply:
Done